src/main/java/com/paneedah/weaponlib/ClientEventHandler.java

```diff
@@ -231,9 +228,6 @@ private void update() {
       
 		modContext.getPlayerItemInstanceRegistry().update(player);
 		final PlayerWeaponInstance mainHandHeldWeaponInstance = modContext.getMainHeldWeapon();
-
-		if (PlayerUtil.isProning(player)) PlayerUtil.slowPlayerDown(player, SLOW_DOWN_WHILE_PRONING_ATTRIBUTE_MODIFIER);
-		else PlayerUtil.restorePlayerSpeed(player, SLOW_DOWN_WHILE_PRONING_ATTRIBUTE_MODIFIER);
```

Comment:
Have you checked if those two methods are used anywhere else?

Reply:
![image](https://github.com/Cubed-Development/Modern-Warfare-Cubed/assets/76911089/6a040c47-333d-411b-bf8f-bef820c66aaa)

Yes. They are used for changing the player speed when scoped.